### PR TITLE
Updated masked image modeling example for Keras 3.

### PR DESCRIPTION
**Pull request by hertschuh**

### CODE REVIEW

**Comment by hertschuh** [comment]
I noticed that the loss is going up during training instead of going down like before. So, I'm not sure what's wrong.

However the MAE metric is going down just like before.

**Review by fchollet — APPROVED** [review]
Yeah, weird numbers. Anyhow, LGTM, thank you!
